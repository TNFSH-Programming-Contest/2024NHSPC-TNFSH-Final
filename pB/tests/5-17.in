41 10
1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1
4296451 978517890 21348516 6 35 4441228 21164 1 1440 205
0 
3 10 8 7
32
0 
0 
21
4 33 7 18 4 7 1 4 2
8 5 9 2 1 10 4 7 8
71
9 20 7 14 9 18 8 5 2 11 7 13 5 40 5 3 7 17 2
0 
29
7 11 5 18 10 24 9 29 6 32 10 9 3 31 7
5 4 1 2 7 10
25
2 7 4 28 4
7 5 9 4 6 8 2 10
95
1 16 3
8 6 7 3 8 2 1 5 9
39
3 19 8 20 7 3 10
5 2 9 6 1 7
0
2 8 2 40 9
8 9 5 6 7 2 4 3 1
29
7 11 8 32 9 6 7 20 7 23 7 8 7 5 10
10 6 5 1 10 4 7 3 9 8 2
97
0 
8 3 4 7 9 10 8 6 1
48
5 4 1 21 2 29 7 3 2 8 4
5 8 3 5 2 1
64
10 26 5 16 9 17 9 32 9 37 5 38 9 22 9 31 8 36 9 4 1
2 1 7
91
2 5 4 4 7
10 4 5 1 6 3 8 7 10 2 9
13
0 
4 3 4 9 1
12
8 30 5 20 2 8 4 15 9 35 1 4 9 6 6 18 6
3 8 7 10
23
6 9 10 39 4 16 10 1 2 41 2 32 2
5 10 1 5 4 3
92
7 10 3 22 5 3 6 16 10 15 8 6 5 25 3
2 1 5
75
1 29 2
5 1 9 6 2 10
15
9 39 5 6 7 12 8 16 3 8 1 37 7 33 8 27 1 28 8
1 7
6
0 
9 5 10 9 2 4 6 8 7 3
70
10 15 6 28 4 22 9 32 2 10 6 25 4 20 7 38 9 30 7 2 10
1 2
31
5 29 6 25 2 27 3 21 9 30 2
10 3 9 5 7 1 6 10 4 2 8
9
7 9 10 29 6 10 1 13 1 21 10 19 2 33 7
0 
22
9 34 2 12 6 10 2 40 1 35 4 39 5 30 2 28 7 29 3
3 10 2 8
56
9 35 4 16 2 33 5 4 6 9 2 8 3 11 10 1 9 38 2
4 4 6 5 7
96
7 25 2 10 6 3 3 16 1 8 10 39 8 29 10
5 9 6 5 1 2
51
1 25 4
0 
15
10 8 1 37 7 11 10 1 8 5 3 26 4 16 7 29 1 19 4 4 10
6 8 2 4 1 6 10
97
3 11 6 35 1 14 10
9 1 8 5 6 10 7 2 9 4
64
0 
9 3 7 4 6 8 1 10 5 9
40
7 27 1 38 8 5 2 6 10 1 6 19 2 2 3
1 10
58
6 38 3 33 3 5 10 16 10 3 2 28 4
0 
19
2 17 8 36 4
5 7 4 5 6 9
61
4 9 8 5 3 24 10 31 6
1 4
69
8 18 7 33 2 38 7 22 3 8 9 10 8 7 8 17 4
2 6 1
74
10 23 8 37 4 3 5 32 6 8 3 1 6 39 10 6 5 28 2 18 2
9 3 5 9 4 7 8 6 10 1
68
4 19 10 4 1 9 6 5 6
0 
94
6 34 3 24 7 17 8 19 9 40 9 16 9
4 5 6 7 2
78
7 33 2 21 6 20 9 10 4 18 10 6 6 36 3
10 6 10 4 5 7 1 8 9 3 2
49
0 
10 1 9 5 6 3 4 2 8 10 7
90
