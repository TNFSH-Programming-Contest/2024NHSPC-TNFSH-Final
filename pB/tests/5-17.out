29851750
4319323
-1
-1
-1
1004330463
-1
-1
-1
-1
1025444212
-1
-1
-1
-1
-1
1029741023
-1
-1
-1
7
-1
4296464
-1
51222888
-1
-1
-1
-1
-1
21371310
-1
-1
-1
1008371247
